src/components/common/select/Select.tsx

```
@@ -34,7 +34,11 @@ export const Select = <TValue, >({
 
     return (<div role={"toolbar"} ref={selectContainerRef}
                  onClick={handleOpenSelect}
-                 className={`${className ?? ''} select ${isOpen ? 'select-opened' : 'select-closed'}`}
+                 className={classNames("select", {
+                    [className!]: className!!,
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)